src/locales/es/interactives/global-climate/global-climate-1.json

```diff
@@ -311,5 +311,7 @@
       "showHeat"
     ],
     "perTick": []
+    "lang": "es",
```

Comment:
You should see an error marked in this online editor. There's a comma missing in line 313 (red marker is a bit misleading, but it's a good indication there's some problem). This kind of errors should be visible when you test things locally too.